### PR TITLE
Control that resolution value is in range

### PR DESCRIPTION
**Pull request by davidmanzanares**
Fix https://github.com/CartoDB/renderer-prototype/issues/209

### CODE REVIEW

File: test/unit/api/style.test.js [hunk 1]
```diff
@@ -79,6 +79,24 @@ describe('api/style', () => {
                 }).toThrowError('`resolution` must be a number.');
             });
 
+            it('should throw an error when resolution is too small', () => {
+                const styleSpec = {
+                    resolution: 0
+                };
+                expect(function () {
+                    new Style(styleSpec);
+                }).toThrowError(/[\s\S]*must.*greater[\s\S]*/g);
```
Comment (Jesus89) [inline test/unit/api/style.test.js]: I like to use the full string exception. `'`resolution` must be greater than $0.'`

File: test/unit/api/style.test.js [hunk 2]
```diff
@@ -79,6 +79,24 @@ describe('api/style', () => {
...
+            });
+
+            it('should throw an error when resolution is too big', () => {
+                const styleSpec = {
+                    resolution: 10000
+                };
+                expect(function () {
+                    new Style(styleSpec);
+                }).toThrowError(/[\s\S]*must.*less[\s\S]*/g);
```
Comment (Jesus89) [inline test/unit/api/style.test.js]: Same here:
'`resolution` must be less than 255.'

File: test/unit/api/style.test.js (hunk 1 above)
Comment (davidmanzanares) [reply test/unit/api/style.test.js]: That repeats the entire error message two times, why coupling so much the test when it is not needed?

Comment (Jesus89) [reply test/unit/api/style.test.js]: The error strings are part of the API, as a function or a namespace. If you change it, the interface you provide to the user changes and we need to detect that the API has changed.

In this case if you change min or max resolution (API) no test fails. Adding the full text this case is also covered.

**Pull request merged**